src/seeders/slots/SlotSeeder.js

```diff
@@ -28,10 +29,10 @@ class SlotSeeder {
 
   static generate(data = {}) {
     return {
-      name: faker.animal.lion(),
-      displayType: faker.random.arrayElement(SlotSeeder.DISPLAY_TYPES),
+      name: RandomDataGenerator.frog(),
+      displayType: RandomDataGenerator.arrayItem(SlotSeeder.DISPLAY_TYPES),
       dataStreams: [],
-      unit: faker.random.arrayElement(Slot.ALL_UNIT_VALUES),
+      unit: RandomDataGenerator.arrayItem(Slot.ALL_UNIT_VALUES || ['a', 'b', 'c']),
```

Comment:
Slot.ALL_UNIT_VALUES does not exist, the faker behaviour is generating random letters, so i had to improvise here

Reply:
If ALL_UNIT_VALUES does not exist, remove it! Instead of a random letters, use these
['°C', 'rpm', '-'] (more realistic)